modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -97,6 +115,34 @@ Returns details for the subscription with ID `<subscription_id>`
 }
 ----
 
+[discrete]
+=== HTTP method
+
+`DELETE api/ocloudNotifications/v1/subscriptions/<subscription_id>`
+
+[discrete]
+=== Description
+
+Deletes the subscription with ID `<subscription_id>`.
+
+.Query parameters
+[cols=2*, width="60%", options="header"]
+|====
+|Parameter
+|Type
+
+|`<subscription_id>`
```

Comment:
Is `<subscription_id>` actually the parameter name? Usually parameter names are exact and not a user-replaced value.Maybe the table could be removed and the string type could be added as a sentence?

Reply:
Hi @jzding 

I noticed in one of the API reference doc sections that parameters as part of the API endpoints have been called "Global path parameters". 
Link - https://78920--ocpdocs-pr.netlify.app/openshift-enterprise/latest/rest_api/console_apis/consoleclidownload-console-openshift-io-v1#apisconsole-openshift-iov1consoleclidownloadsname

So can we update this API end point as follows:
"DELETE api/ocloudNotifications/v1/subscriptions/{subscription_id}`"

And change the table name from "Query parameters" to "Global path parameters"?

Reply:
yes {subscription_id} should be Global path parameters

Reply:
same applies to `resource_address` in api/ocloudNotifications/v1/<resource_address>/CurrentState

Reply:
Hi @dfitzmau , @jzding 
Updated as per the discussion above.
Thanks!